scripts/src/java/org/oppia/android/scripts/common/RepositoryFile.kt

```diff
@@ -5,39 +5,41 @@ import java.io.File
 /** Helper class for managing & accessing files within the project repository. */
 class RepositoryFile() {
   companion object {
-    /** a list of directories and files which should be excluded for every script check. */
-    val alwaysExcludeFilesList = listOf<String>(
+    /** A list of directories which should be excluded for every script check. */
+    private val alwaysExcludeDirectoryList = listOf<String>(
       ".git",
       ".gitsecret",
       ".idea",
-      "bazel",
-      "config",
-      "gradle",
-      ".bazelrc",
-      ".editorconfig",
-      "gradlew",
-      "gradlew.bat",
       ".aswb",
+      "gradle",
+      "bazel-bin",
+      "bazel-oppia-android",
+      "bazel-out",
+      "bazel-testlogs",
     )
 
     /**
      * Collects the paths of all the files which are needed to be checked.
+     * Files that are generated or can't be altered in style/best practices (such as those managed
+     * by Android Studio) are automatically exempted.
      *
      * @param repoPath the path of the repo
-     * @param allowedExtension files with only this extension will be included in the search list
-     * @param exemptionsList a list of files which should not be included in the search list
-     * @return all files which needs to be checked
+     * @param expectedExtension files with only this extension will be included in the search list.
+     * This defaults to the empty string which signifies no extension restriction.
```

Comment:
Multi-line at-param continuation is +4 spaces from the '@' symbol.

Reply:
Noted. Done